content/infrastructure/installation/install-rippled-on-ubuntu.md

```diff
@@ -91,6 +89,28 @@ Before you install `rippled`, you must meet the [System Requirements](system-req
         sudo setcap 'cap_net_bind_service=+ep' /opt/ripple/bin/rippled
 
 
+9. Optional: configure core dumps
+
+    By default Ubuntu is not configured to produce core files useful for debugging crashes.
+    First run:
+    
+            ulimit -c unlimited
+    
+    Now run `sudo systemctl edit rippled`. The default editor should open and add
+
+        [Service]
+        LimitCORE=infinity
+
+    This creates `/etc/systemd/system/rippled.service.d/override.conf` saving core dumps while not interfering 
+    with the service file provided by the `rippled` package.
+    Core dumps appearing in `/var/lib/apport/coredump/` can be loaded into `gdb` with
+
+        gdb <path_to_rippled> <path_to_core_file>
```

Comment:
Rather than using `<path_to...>` which may be interpreted as shell syntax, I prefer to use examples that use reasonable values. On Ubuntu, we know the path to the rippled executable, so that's a good start; if you had an example of a full core dump filename that would be even better, but I think this would be an improvement:

```suggestion
        gdb /opt/ripple/bin/rippled /var/lib/apport/coredump/core
```